adapters/tappx/params_test.go

```diff
@@ -60,4 +65,11 @@ var invalidParams = []string{
 	`{"tappxkey": 1, "endpoint":"ZZ1INTERNALTEST149147915", "host":""}`,
 	`{"tappxkey":"pub-12345-android-9876", "endpoint": 1, "host":""}`,
 	`{"tappxkey": 1, "endpoint": 1, "host": 123}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
+	`{"tappxkey": "1", "endpoint": "1", "mktag":1}`,
+	`{"tappxkey": "1", "endpoint": "1", "mktag":[1,2]}`,
+	`{"tappxkey": "1", "endpoint": "1", "host":"host", "bcid":""}`,
+	`{"tappxkey": "1", "endpoint": "1", "host":"host", "bcid":"123", bcrid: ["123"]}`,
+	`{"tappxkey": "1", "endpoint": "1", "host":"host", "bcid":["123"], bcrid: 123}`,
```

Comment:
Could you please add invalid test cases for `bcid` and `bcrid` where the array item type is invalid? Something like: .`[123]`. This will ensure json schema is property enforcing the array type constraint.

Reply:
Done in 9e3aff2 and 9636101 commits.